src/ansys/tools/versioning/utils.py

```diff
@@ -139,8 +139,10 @@ def server_meets_version(server_version, required_version):
 
     Parameters
     ----------
-    server_version : str, tuple
+    server_version : str, tuple, obj
         A string or tuple representing the server version.
+        If it is an object, it must have a '_server_version' attribute.
+
```

Comment:
This docstring is ill-formed... strings or tuples are also objects. We should be a bit more concise.
```suggestion
    server_version : str, tuple, obj
        A string or tuple representing the server version.
        If it is an object different from the previous ones, it must have a '_server_version' attribute.

```